backend/backend.js

```diff
@@ -45,6 +46,12 @@ app.get("/pins", async (req, res) => {
   }
 });
 
+app.get("/pins/:id", async (req, res) => {
```

Comment:
We made drastic changes to the get function. Can you pull from main and refactor the code to match it?

Reply:
Nevermind.